flytekit/remote/remote.py

```diff
@@ -123,6 +124,9 @@ def from_config(
         :param default_project: default project to use when fetching or executing flyte entities.
         :param default_domain: default domain to use when fetching or executing flyte entities.
         """
+        # TODO: flyte-cli already support users to set different flyte config file path, we should
```

Comment:
could you help with this after this PR goes in?

Reply:
sure